### DISCUSSION TITLE
Reuse css variable when using semantic token with refference

### DISCUSSION THREAD

**Discussion by isBatak** [chakra-ui/panda#1196] [Ideas]
### Description

Current implementation duplicates the color value for referenced semantic tokens.

For example:
```js
const tokens = {
  colors: {
    red: {
      value: '#ef4444',
    },
  },
  semanticTokens: {
    colors: {
      danger: { value: '{colors.red}' },
    },
  },
}
```
will output
```css
--colors-red: #ef4444;
--colors-danger: #ef4444;
```

### Problem Statement/Justification

Seems like a logical and reasonable implementation.
If we forcefully override panda css variable in our project that change will be also applied to semantic token.

### Proposed Solution or API

we should reference the original css var like this:
```css
--colors-red: #ef4444;
--colors-danger: var(--colors-red);
```

**Comment by segunadebayo** [comment]
Converting this to a discussion since it's technically not a bug but more of a feature-request.

Thanks for working to implement this.

**Comment by ivanbanov** [comment]
this is a must-have, fully reactive themes need it, in the meantime the escape hatch is `var(--colors-foo)`

**Comment by tanangular** [comment]
My workaround solution is use with token.var()
```
tokens: {
  colors: {
    red: { value: '#ef4444'},
  }
},
semanticTokens: {
  colors: {
    danger:{ value: `${token.var('colors.red')}` }
  }
}

/*
--colors-red: #ef4444;
--colors-danger: var(--colors-red);
*/
```
